tests/testthat/test-verb-pivot-longer.R

```diff
@@ -84,6 +84,31 @@ test_that("can override default output column type", {
   )
 })
 
+test_that("`values_transform` works with single functions (#1284)", {
```

Comment:
Miiiight be worth similar tests for `names_ptypes` and `names_transform`. Those are obviously a little different because they are handled by `tidyr::build_longer_spec()` directly, but still may be good to validate that it is working right.

Reply:
I added one test to see that they are passed through correctly.